src/main/webapp/web.xml

```diff
@@ -3,6 +3,9 @@
          xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee
                       http://xmlns.jcp.org/xml/ns/javaee/web-app_4_0.xsd"
          version="4.0">
+    <session-config>
+        <session-timeout>1</session-timeout>
```

Comment:
Our session end in one minute it's very quick

Reply:
It's very secure taxi service which carry most valuable persons. So we need to be sure it's really secure))